lua/dylib.lua

```diff
@@ -85,28 +101,19 @@ function print_fib_dump_entry(fib_dump_entry, acc)
 
 	ip_addr_str = dylib.ip_format_addr(entry.addr)
 	acc = acc .. "FIB entry for IP prefix: " .. ip_addr_str ..
-		"/" .. entry.prefix_len .. " with action " ..
-		tostring(entry.action)
+		"/" .. entry.prefix_len .. " with action " .. entry.action
```

Comment:
If we removed the ``tostring(entry.action)``, then there is an error: ``./lua/dylib.lua:104: cannot convert 'string' to 'enum gk_fib_action'!``

Reply:
Alright, you can put the `tostring()` back. Thanks for testing things up.